common/app/views/fragments/nav/newHeaderMenu.scala.html

```diff
@@ -29,11 +29,12 @@
     </li>
 }
 
-<div class="main-menu-container" id="main-menu">
+<div href="#" class="main-menu-container__overlay" aria-hidden="true"></div>
+
+<div class="main-menu-container">
         <!-- TODO: Provide tabbable alternative -->
-    <a href="#" class="main-menu-container__overlay" aria-hidden="true"></a>
 
-    <div class="main-menu-container__menu js-reset-scroll-on-menu">
+    <div class="main-menu-container__menu js-reset-scroll-on-menu" id="main-menu" aria-hidden="true">
```

Comment:
I've updated the class name to `js-main-menu`. The JavaScript that does the scrolling now looks like:

```
var menuContainer = qwery('.js-main-menu')[0];

menuContainer.scrollTop = 0;
```